backend/app/deps/authorization_deps.py

```diff
@@ -92,6 +92,41 @@ async def __call__(
             )
 
 
+class DatasetAuthorization:
```

Comment:
The base Authorization class that Luigi implemented functions as DatasetAuthorization, you should be able to remove this one and use Authorization instead.

Reply:
Removed.